floor_generator/pubspec.yaml

```diff
@@ -17,8 +17,7 @@ dependencies:
   build_config: ^1.0.0
   code_builder: ^4.1.0
   collection: ^1.15.0
-  floor_annotation:
-    path: ../floor_annotation
```

Comment:
For development purposes we should point these to the local one, so these should not be changed

Reply:
Yeah, I see it is comfortable to use relative imports in the development stage, but looking forward, in terms of using the plugin with my version until the next version comes out, it is better to change that coz there is not way to import  plugin from the GIT directly because the relative imports.

Reply:
> For development purposes we should point these to the local one, so these should not be changed

Yeah, I see it is comfortable to use relative imports in the development stage, but looking forward, in terms of using the plugin with my version until the next version comes out, it is better to change that coz there is no way to import plugin from the GIT directly because the relative imports.